content/events/2020-london/contact.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Contact
+---
+
+If you’d like to contact us by email: organisers@kubernetesdays.london
+
```

Comment:
Can you use the KCD provided email address organizers-london@kubernetescommunitydays.org here?